src/Files.Uwp/ServicesImplementation/Settings/ApplicationSettingsService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Files.Backend.Services.Settings;
+using Files.Shared.Enums;
+using Files.Uwp.Serialization;
+
+namespace Files.Uwp.ServicesImplementation.Settings
+{
+    internal sealed class ApplicationSettingsService : BaseObservableJsonSettings, IApplicationSettingsService
+    {
+        public bool WasPromptedToReview
```

Comment:
I think like this the setting will be included when exporting settings. Not sure if it's a problem. And I'm not sure if/how we esclude some settings from the import/export.

Reply:
That's good, we don't want to prompt multiple times.